cmd/cpr/pending_review.go

```diff
@@ -18,7 +18,7 @@ import (
 // PendingReview analysis of open pull requests
 func PendingReview(token string, dryRun bool, owner string, repo string) error {
 	context := context.Background()
-	client, err := internal.MakeClient(context, token)
+	client, err := internal.MakeClient(context, token, pending_review.TargetRepository{Owner: owner, Name: repo})
```

Comment:
I am not sure the name for this 🤔 

Is it the "target where we save" is it the "source we are working from" ... maybe just "working" 😖